library/core/src/ptr/const_ptr.rs

```diff
@@ -966,16 +975,16 @@ impl<T> *const [T] {
     ///     * The entire memory range of this slice must be contained within a single allocated object!
     ///       Slices can never span across multiple allocated objects, and
     ///
-    ///     * The pointer must be aligned even for zero-length slices. One
+    ///     * the pointer must be aligned even for zero-length slices. One
```

Comment:
"Slices can never span across multiple allocated objects, and the pointer must be aligned even for zero-length slices." -- that's an odd sentence, so I think this is worth changing to proper sentences as well.

(I hope I am not nitpicking too much here, if you want we can also land this and do a cleanup/editing PR afterwards.)

Reply:
That review is already outdated ;)

Reply:
Ah, I got no further email from GH but I guess you force-pushed?

Reply:
No, I think your diff was outdated. The changes where pushed with 82b8d75 :)
I try not to force push whenever possible.